Update protobufs

xPaw opened the pull request:
- Updated ip field names to use deprecated ones. See #758
- Legacy servicemethod
- `CMsgClientServerList` was removed - https://github.com/SteamDatabase/Protobufs/commit/a4ba0a9e35a204b8185ca32a8aec697c830cadb2#diff-0f28215af4ff6780216ed68eec25ff52

yaakov-h commented:
So server-list-via-CM is dead now?

yaakov-h commented on Resources/NetHookAnalyzer2/NetHookAnalyzer2/Specializations/CSGO/CSGOSOMultipleObjectsGCSpecialization.cs:
```diff
@@ -3,7 +3,8 @@
 using ProtoBuf.Meta;
-using SteamKit2.GC.CSGO.Internal;
+//using SteamKit2.GC.CSGO.Internal;
+using SteamKit2.GC.Dota.Internal; // CSGO no longer has objects_added and objects_removed
```
if CS:GO no longer has those fields, should we just delete the code in this class that handles them? Or is data still there but it just has no field in the protos?

yaakov-h replied:
Last issue outstanding.

xPaw commented:
I removed `GetServersOfType` which means `CDNClient.FetchServerListAsync` also had to be removed. That method is trivially replaced by `ContentServerDirectoryService.LoadAsync`, like depot downloader already did: https://github.com/SteamRE/DepotDownloader/commit/c6975323d2915a451119260ded45032a47e4fa47

`UFSClient` also used the server list, I changed it to require a server for now.
